assets/css/bootstrap/_map_overrides.scss

```diff
@@ -11,3 +11,14 @@ $theme-colors: (
   "dark": new_tokens.$gray-900,
   "missed-stop": semantic.$missed-stop,
 );
+
+$utilities: (
+  // Shorthand for specifying utility classes `top-0 start-0 bottom-0 end-0`
```

Comment:
I'm curious about this comment - are `top-0`, etc examples of possible utility classes?

Reply:
I ask because unless I'm missing something special about the `$utilities` syntax, we're only actually short-handing `inset`?